src/client/renderingengine.cpp

```diff
@@ -179,7 +179,10 @@ RenderingEngine::RenderingEngine(MyEventReceiver *receiver)
 
 	// bpp, fsaa, vsync
 	bool vsync = g_settings->getBool("vsync");
-	bool enable_fsaa = g_settings->get("antialiasing") == "fsaa";
+	// Don't enable MSAA in OpenGL context creation if post-processing is enabled,
+	// the post-processing pipeline handles it.
+	bool enable_fsaa = g_settings->get("antialiasing") == "fsaa" &&
+			!g_settings->getBool("enable_post_processing");
```

Comment:
This has the nice side effect that you can set really high MSAA values without getting blurry fonts or GUIs

Reply:
Indeed!